src/pdist/problem_description.cpp

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+#include <sstream>
+
+#include <miopen/pdist/problem_description.hpp>
+#include <miopen/names.hpp>
+
+namespace miopen {
+
+namespace pdist {
+
+NetworkConfig BackwardProblemDescription::MakeNetworkConfig() const
+{
+    auto input_dtype = inputDesc.GetType();
+    auto input_dims  = inputDesc.GetLengths();
+
+    std::ostringstream ss;
+    ss << "pdist_backward";
+    ss << "dtype" << input_dtype;
+    ss << "dims" << input_dims[0] << "x" << input_dims[1];
+    ss << "p" << p;
```

Comment:
this is redundant because its not appear in IsApplicable

Reply:
I'm not fully familiar with the `MakeNetworkConfig()`, so just to make sure, I want to confirm that the redundancy is specifically related to the include of `p`, and that removing line 44 would resolve the issue completely, is this correct?

Reply:
resolved: remove p since its not included in `IsApplicable()` or `GetSolution()`